deploy/proxy_deploy.yaml

```diff
@@ -16,5 +16,10 @@ spec:
     spec:
       serviceAccountName: proxy-controller
       containers:
-      - name: main
-        image: proxy-controller
+        - name: main
+          image: proxy-controller
+          env:
+            - name: NAMESPACE
```

Comment:
does the proxy only work for 1 namespace?

Reply:
Yes, you would need 1 proxy per namespace as currently setup. We can introduce a cluster-scoped proxy in the future.